backend/src/main/java/com/google/models/MetricValue.java

```diff
@@ -27,6 +27,10 @@ public MetricValue(int value) {
     this.value = value;
   }
 
+  public MetricValue(long value) {
+    this.value = (int) value;
```

Comment:
is this safe? quick Google search came up with this: https://stackoverflow.com/questions/1590831/safely-casting-long-to-int-in-java
consider using this ^?

Reply:
fixed it! thanks for the suggestion :)